torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -648,6 +648,11 @@ void MapXlaEnvVarsToLazy() {
       runtime::sys_util::GetEnvInt("XLA_TRIM_GRAPH_SIZE", 100000);
 }
 
+at::Tensor MarkTensor(const at::Tensor& input, const std::string& info) {
+  auto result = tensor_methods::mark_tensor(bridge::GetXlaTensor(input), info);
```

Comment:
lol call me anti-auto, but can we explicitly spell out he basic types? It will make code reading a lot easier.

Reply:
Makes sense, let me explicitly specify the type

Reply:
Done